webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5080,7 +5080,7 @@ var MCK_BOT_MESSAGE_QUEUE = [];
                 var messageClass= "vis";
                 var progressMeterClass = "n-vis";
                 var attachmentBox = "n-vis";
-                if(msg && msg.metadata && msg.metadata.feedback){
+                if((msg && msg.metadata && msg.metadata.feedback) || msg.metadata.KM_ASSIGN_TO){ // KM_ASSIGN_TO parameter comes when we change assigne by bot message.
```

Comment:
@ab14bhardwaj correct spelling for the assignee

Reply:
@devashishmamgain updates the spelling of the assignee.